<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1583141599368' stop='1583141604100'>
    <name>API Testing</name>
    <title>API Testing</title>
    <test-cases>
        <test-case start='1583141599369' status='failed' stop='1583141600270'>
            <name>API:GET CALL @smoke</name>
            <title>API:GET CALL @smoke</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='6c25babc-c80c-4151-ae1d-392dcefce41b-attachment.png' type='image/png' size='3249'/>
            </attachments>
            <failure>
                <message>Expected 'VCA-OU-71' to equal 'VCA-OU-72'.</message>
                <stack-trace>Error: Failed expectation
                    at UserContext.&lt;anonymous> (D:\Mars_VoyagerAPI\TestSuites\sampleTestCase.spec.js:65:38)
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1583141600270' status='failed' stop='1583141602184'>
            <name>API:POST CALL @smoke</name>
            <title>API:POST CALL @smoke</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='566e990e-f31d-4ff4-aa2d-57179634fd05-attachment.png' type='image/png' size='3249'/>
            </attachments>
            <failure>
                <message>Expected 'VCA-OU-170' to equal 'VCA-OU-72'.</message>
                <stack-trace>Error: Failed expectation
                    at UserContext.&lt;anonymous> (D:\Mars_VoyagerAPI\TestSuites\sampleTestCase.spec.js:101:37)
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1583141602184' status='passed' stop='1583141603193'>
            <name>API:PUT CALL @regression</name>
            <title>API:PUT CALL @regression</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='daf9c151-1ed0-4880-bd7b-fc219a7277d9-attachment.png' type='image/png' size='3249'/>
            </attachments>
        </test-case>
        <test-case start='1583141603193' status='failed' stop='1583141604099'>
            <name>API:GET CALL FOR NOTES @smoke</name>
            <title>API:GET CALL FOR NOTES @smoke</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='05f7f5b9-ad30-4fe0-acaf-dcd844e00d29-attachment.png' type='image/png' size='3249'/>
            </attachments>
            <failure>
                <message>Expected Object({ id: 117, accountNumber: 'VCA-OU-72', createdBy: 'Manoj', updatedBy: null, note: 'This note is created' }) to equal 'VCA-OU-72'.</message>
                <stack-trace>Error: Failed expectation
                    at UserContext.&lt;anonymous> (D:\Mars_VoyagerAPI\TestSuites\sampleTestCase.spec.js:137:37)
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>